src/python/cli_new/lib/cli/http.py

```diff
@@ -41,6 +41,8 @@ def read_endpoint(addr, endpoint):
 
     try:
         url = "{addr}/{endpoint}".format(addr=addr, endpoint=endpoint)
+        if query is not None:
+          url += "?{query}".format(query=urllib.parse.urlencode(query))
```

Comment:
lib/cli/http.py:45:0: W0311: Bad indentation. Found 10 spaces, expected 12 (bad-indentation)

Reply:
I fixed all the linting, and added a commit to fix an optional parameter ordering issue.

However I was not able to pass tests before my changes.

Reply:
Checking again, our timing was quite unfortunate today 😄.